incubator/solr/solr.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# Docker usage: https://hub.docker.com/_/solr/
```

Comment:
Yeah, i think i should take these out.  It was really just my own notes on where i got the image from.  I think it is easy enough to just look at the image and know it was from docker hub.